app/models/certification.rb

```diff
@@ -17,21 +17,16 @@ class Certification < ApplicationRecord
   belongs_to :claim, class_name: 'Claim::BaseClaim'
   belongs_to :certification_type
 
+  validates :certification_type_id, presence: true, if: :claim_agfs?
```

Comment:
This should ideally follow the existing validator pattern that uses `CertificationValidator` 

also, though not from this PR,  but i think the `claim_agfs?` is out of date or wrong. I think all agfs claims may require this, but need to check that with the business. see related comment below